Source/WTF/wtf/text/AtomString.h

```diff
@@ -111,12 +111,12 @@ class AtomString final {
 
 #if USE(CF)
     AtomString(CFStringRef);
-#endif
 
 #ifdef __OBJC__
     AtomString(NSString *);
     operator NSString *() const { return m_string; }
 #endif
+#endif
```

Comment:
This doesn't look right.  If you want this code to be available only on Apple platforms, you should probably use `OS(DARWIN)` around both `#if USE(CF)` and `#ifdef __OBJC__`.